app/services/problem_check/ai_llm_status.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class ProblemCheck::AiLlmStatus < ProblemCheck
+  self.priority = "high"
+  self.perform_every = 6.hours
+
+  def call
+    llm_errors
+  end
+
+  def base_path
+    Discourse.base_path
+  end
+
+  private
+
+  def llm_errors
+    return [] if !SiteSetting.discourse_ai_enabled
+    LlmModel.in_use.find_each.filter_map do |model|
+      try_validate(model) { validator.run_test(model) }
+    end
+  end
+
+  def try_validate(model, &blk)
+    begin
+      blk.call
+      nil
+    rescue => e
+      error_message = parse_error_message(e.message)
+      message =
+        "#{I18n.t("dashboard.problem.ai_llm_status", { base_path: base_path, model_name: model.display_name, model_id: model.id })}"
```

Comment:
Sorry. Late to the party. This shouldn't really need to be wrapped in an interpolation string? 🙂